andela/travel_tool_front
# # 161968220 Adds cancel button to mobile navigation

#### What does this PR do?
Adds the cancel button to the side navigation bar on the mobile view of the app

#### Description of Task to be completed?
Currently, when the user views the mobile navigation, it does not have a `close button` that closes it. This PR introduces this `cancel/close` button that allows users to close the side navigation on the mobile view

#### How should this be manually tested?
- Clone the repo
    `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory
   `cd travel_tool_front`
 - Checkout the **ch-mobilenav-close-161968220** branch
   `git checkout ch-mobilenav-close-161968220`
 - Install dependencies and start the server
   - `make start` *if you have docker*
   - `yarn install` then `yarn start` *if you do not have docker*
 - Start the [backend](https://github.com/andela/travel_tool_back) of this project.
 - Navigate to [travela](http://travela-local.andela.com:3000/) in your browser and log in with your Andela account. 
- Navigate to [travela requests](http://travela-local.andela.com:3000/requests) in your browser.
- While on `Inspect Mode`, click on the toggle view button (top right corner of the developer console) to switch to the mobile view
<img width="70" alt="screen shot 2018-12-13 at 12 27 35" src="https://user-images.githubusercontent.com/31407254/49929207-0299f980-fed3-11e8-80c1-0b222cd4279c.png">

- Click on the three lines stacked on top of each other (menu bar)
<img width="399" alt="screen shot 2018-12-13 at 12 32 17" src="https://user-images.githubusercontent.com/31407254/49929348-586ea180-fed3-11e8-9ddc-1b55a3a0cf87.png">

- You should see the mobile navigation bar sliding from the left
- It should have a cancel button that closes it

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161968220
Story is [here](https://www.pivotaltracker.com/story/show/161968220)

#### Screenshots (if appropriate)
<img width="404" alt="screen shot 2018-12-13 at 11 53 15" src="https://user-images.githubusercontent.com/31407254/49929180-ebf3a280-fed2-11e8-805c-f5bd974f94d0.png">

#### Questions:
N/A
